### PR TITLE
Add Github Actions badge

### PR DESCRIPTION
**Pull request by ashumkin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Keepalived Exporter
 Prometheus exporter for [Keepalived](https://keepalived.org) metrics.
 
+[![Continuous Integration](https://github.com/cafebazaar/keepalived-exporter/workflows/Continuous%20Integration/badge.svg)](https://github.com/cafebazaar/keepalived-exporter/actions)
```
Comment (clwluvw) [inline README.md:4]: Thanks!
I think it would be better to go under "Keepalived Exporter" title, upper than description.

Comment (ashumkin) [reply README.md:4]: > Thanks!
> I think it would be better to go under "Keepalived Exporter" title, upper than description.

Agree )

**Comment by clwluvw** [comment]
@ashumkin Why did you close this PR?

**Comment by clwluvw** [comment]
> > @ashumkin Why did you close this PR?
> 
> I did not... I don't know why it is marked as I did

It's okay :wink: 
Can you please change your commit to pass semantic pull Request too?

**Pull request merged**